genai/example_test.go

```diff
@@ -378,6 +418,70 @@ func ExampleGenerativeModel_GenerateContentStream_videoPrompt() {
 
 }
 
+func ExampleGenerativeModel_GenerateContentStream_audioPrompt() {
+	ctx := context.Background()
+	client, err := genai.NewClient(ctx, option.WithAPIKey(os.Getenv("GEMINI_API_KEY")))
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.Close()
+
+	model := client.GenerativeModel("gemini-1.5-flash")
+
+	file, err := uploadFile(ctx, client, filepath.Join(testDataDir, "sample.mp3"), "")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.DeleteFile(ctx, file.Name)
+
+	iter := model.GenerateContentStream(ctx,
+		genai.Text("Give me a summary of this audio file."),
+		genai.FileData{URI: file.URI})
+	for {
+		resp, err := iter.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			log.Fatal(err)
+		}
+		printResponse(resp)
+	}
+
+}
+
+func ExampleGenerativeModel_GenerateContentStream_pdfPrompt() {
+	ctx := context.Background()
+	client, err := genai.NewClient(ctx, option.WithAPIKey(os.Getenv("GEMINI_API_KEY")))
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.Close()
+
+	model := client.GenerativeModel("gemini-1.5-flash")
+
+	file, err := uploadFile(ctx, client, filepath.Join(testDataDir, "test.pdf"), "")
```

Comment:
Why doesn't this use client.UploadFileFromPath?

Reply:
ditto below

Reply:
Haven't rebased all the PRs yet. I will fix it when landing the other one